qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -211,11 +212,13 @@ def search(self, text, *, ignore_case='never', reverse=False,
         if reverse:
             self._flags |= QWebEnginePage.FindBackward
 
+        self._tab.scripts.update_stylesheet(searching=True)
         self._find(text, self._flags, result_cb, 'search')
 
     def clear(self):
         self.search_displayed = False
         self._widget.findText('')
+        self._tab.scripts.update_stylesheet(searching=False)
 
     def prev_result(self, *, result_cb=None):
```

Comment:
Maybe we should add the bar when using next/prev_result as well, that would cause using `n` when not searching to pop open the bar. I'm guessing we want the bar whenever the search selection is 'orange'

Reply:
That's a good point, it looks like I can make that happen by just putting `self._tab.scripts.update_stylesheet` into `_find`, so I'll go ahead and do that.